src/app/record/components/top-bar/top-bar.component.ts

```diff
@@ -36,18 +39,32 @@ export class TopBarComponent implements OnInit, OnDestroy {
   creditName = ''
   expandedContent = false
   recordWithIssues: boolean
+  justRegistered: boolean
+  emailVerified: boolean
+  checkEmailValidated: boolean
+  inDelegationMode: boolean
   @Input() loadingUserRecord = true
 
   constructor(
+    private _dialog: MatDialog,
     private _platform: PlatformInfoService,
     private _user: UserService,
-    private _record: RecordService
+    private _record: RecordService,
+    private _recordEmails: RecordEmailsService,
+    private _verificationEmailModalService: VerificationEmailModalService
   ) {
     _platform
```

Comment:
We will need to add the onDesroy to this _platform subscription and many others around the app. So I will create a separate card this those.

Reply:
https://trello.com/c/UGj8ayuJ/7655-qa-devsmy-orcid-subscriptions-clean-up

Reply:
OK, makes sense